lib/bibliothecary/parsers/rubygems.rb

```diff
@@ -10,16 +10,19 @@ class Rubygems
 
       def self.mapping
         {
-          /^Gemfile$|^gems\.rb$|.*\/Gemfile$|.*\/gems\.rb$/ => {
+          # /^Gemfile$|^gems\.rb$|.*\/Gemfile$|.*\/gems\.rb$/ => {
+          lambda { |p| match_filename("Gemfile")[p] || match_filename("gems.rb")[p] } => {
```

Comment:
Could add a match_filenames, or take a list and [filenames].flatten it?

Reply:
I stared at the [p]s for minutes trying to figure out what was happening. I had no idea you could use lambdas like that!

Reply:
Yeah, I wasn't sure about making `match_filenames` for just these few cases

Reply:
I think it's worth adding `match_filenames` for clarity even though only used a few times